sections/placeholders.md

```diff
@@ -0,0 +1,117 @@
+
+##### Optional Parameters:
+
+| **Parameter** | **Description** |
+| ------------- | --------------- |
+| fields | A comma separated list of additional fields to include in the response [ "assignments", "custom_field_values"] |
+| per_page, page | Parameters for pagination. Default values are per_page = 20 , page = 1 ( the first ) |
+
+```
+GET  /api/v1/placeholder_resources/<placeholder_resource_id>
+ curl 'https://vnext.10000ft.com/api/v1/placeholder_resources/12345?fields=assignments&auth=...'
+```
+
+## Create a Placeholder Resource
+
+##### Required Parameters
+
+| `title` |
+
+```
+POST  /api/v1/placeholder_resources
+ curl -d 'title=Designer' \
+             'https://vnext.10000ft.com/api/v1/placeholder_resources?auth=...'
+```
+
+Placeholders are referred to by title. The title is displayed wherever placeholders appear in the application e.g. on projects, the schedule, etc. Therefore, it is required to specify this parameter.
+
+##### Optional Parameters:
+
+| **Parameter** | **Description** |
+| ------------- | --------------- |
+| role | The placeholder's [role](sections/roles.md) in the organization. |
+| discipline | The placeholder's [discipline](sections/disciplines.md). |
+| location | The organizational location the placeholder belongs to. |
+
+##### Parameters not exposed through the API
+
+In the application's user interface, there are certain options available when creating and editing placeholder resources. For example, if a role and discipline are selected, the placeholder title is autopopulated as a combination of discipline and role. Options also exist to upload a thumbnail image for the placeholder, or to choose a color and abbreviation and then auto-generate a thumbnail. These options are currently not available through the API, and can only be used via the front-end UX.
+
+## Update a Placeholder
+
+Placeholder specified by `id`
+
+```
+PUT  /api/v1/placeholder_resources/<placeholder_resource_id>
+ curl -XPUT -d 'title=Senior Designer' \
+             'https://vnext.10000ft.com/api/v1/placeholder_resources/12345?auth=...'
+```
+
+## Delete a Placeholder
+
+Placeholder resources may be deleted via the API, as follows.
+
+```
+DELETE  /api/v1/placeholder_resources/<placeholder_resource_id>
+ curl -XDELETE \
+             'https://vnext.10000ft.com/api/v1/placeholder_resources/12345?auth=...'
+```
+
+## Placeholders and Users/People
+
+As noted above, placeholders behave similarly to real users. However, there are some differences. Here we discuss ways in which placeholders can be used similar to users in the API, and ways they are different.
+
+##### Placeholder Assignments
+
+Assignments can be made, updated, and removed via the [Assignments](sections/assignments.md) API just as they can for users. 
+The only difference is that this is accessed through the placeholder_resources endpoint. In general, for the [Assignments](sections/assignments.md) API, a placeholder resource ID can be used as the user_id parameter.
```

Comment:
I'd put backticks around `placeholder_resources` and `user_id` so they display as code.